addons/killtracker/XEH_postInit.sqf

```diff
@@ -136,9 +136,18 @@ GVAR(killCount) = 0;
                 _unitName = format ["*AI* - %1", getText ((configOf _unit) >> "displayName")];
             };
         };
-        if (_unitIsPlayer || GVAR(trackAI)) then {
-            TRACE_3("send kill event",_killer,_unitName,_killInfo);
-            [QGVAR(kill), [_unitName, _killInfo], _killer] call CBA_fnc_targetEvent;
+        TRACE_3("send kill event",_instigator,_unitName,_killInfo);
+        [QGVAR(kill), [_unitName, _killInfo], _instigator] call CBA_fnc_targetEvent;
+
+        if (GVAR(showCrewKills) && {!(_killer isKindof "CAManBase")}) then {
+            private _crew = [driver _killer, gunner _killer, commander _killer] - [_instigator];
+            _crew = _crew select {[_x] call EFUNC(common,isPlayer)};
+            _crew = _crew arrayIntersect _crew;
+            TRACE_1("showCrewKills",_crew);
+            _killInfo = format [" - [<t color='#99ff99'>%1</t>, %2", localize "str_a3_rscdisplaygarage_tab_crew", _killInfo select [4]];
```

Comment:
I haven't looked at it in detail, but `_killInfo select [4]` is very specific. Can`_killInfo` be `""`? If so, what happens?

Reply:
this is only reachable if line 68 also ran
also `"" select [4]` is safe and just returns `""`